micrometer-tracing/src/main/java/io/micrometer/tracing/Span.java

```diff
@@ -333,6 +387,30 @@ public Span start() {
          */
         Builder tag(String key, String value);
 
+        /**
+         * Sets a tag on the span.
+         * @param key tag key
+         * @param value tag value
+         * @return this
+         */
+        Builder tag(String key, long value);
```

Comment:
This should be `default` and delegate to the String version. Otherwise, this is a breaking change